tests/foreman/ui/test_host.py

```diff
@@ -868,6 +868,89 @@ def test_positive_search_by_parameter(session, module_org, smart_proxy_location,
         assert values[0]['Name'] == param_host.name
 
 
+@pytest.mark.tier2
+@pytest.mark.rhel_ver_match('8')
+@pytest.mark.no_containers
+def test_positive_search_by_reported_data(
+    target_sat, rhel_contenthost, module_org, module_ak_with_cv
+):
+    """
+    Search for host by reported data.
+    For example, search by `reported.bios_vendor = SeaBIOS`.
+
+    :id: 54341d00-34bc-11ef-a8a6-000c29a0e355
+
+    :expectedresults: Return only hosts matching the reported data.
+
+    :Verifies: SAT-9132
+
+    :customerscenario: true
+    """
+    result = rhel_contenthost.register(module_org, None, module_ak_with_cv.name, target_sat)
+    assert result.status == 0, f'Failed to register host: {result.stderr}'
+    client = rhel_contenthost
+
+    reported_data_params = [
+        'bios_release_date',
+        'bios_vendor',
+        'bios_version',
+    ]
+    source_host = target_sat.api.Host().search(query={'search': f'name={client.hostname}'})[0]
```

Comment:
Can we use nailgun_host property here instead?

Reply:
Updated. Didn't know about this handy property.